runtime/doc/options.txt

```diff
@@ -5571,6 +5571,15 @@ A jump table for the options with a short description can be found at |Q_op|.
 	NOTE: This option is set to the Vi default value when 'compatible' is
 	set and to the Vim default value when 'compatible' is reset.
 
+                                                *'relativenumberformat'* *'rnfm'*
+'relativenumberformat' 'rnfm'	string	(default "%-*ld ")
+                        global
+                        {not in Vi}
+        Format the ":relativenumber" command output.
+        This is a scanf-like string.  For example, if you want to
```

Comment:
I don't like this pull request at all (for one thing, because it only incorporates custom formatting for `relativenumber`, not for `number`), but with regard to your comment here I have to be fair: Bram calls it `scanf-like` for the `grepformat` option.

Reply:
`grepformat` is used for scanning the output of `grepprg`, however this is used for printing the line number. So `printf-like` is suitable here.

Reply:
Right! Perhaps it's a good idea to put a tag to `printf()`, because otherwise the documentation is not self contained (not everyone using vim will be familar with `printf`).